x/auth/client/cli/sign.go

```diff
@@ -13,13 +12,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
 	"github.com/spf13/cobra"
-	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/go-amino"
 )
 
 const (
 	flagAppend    = "append"
 	flagPrintSigs = "print-sigs"
 	flagOffline   = "offline"
+	flagSigOnly   = "print-signature-only"
```

Comment:
what's the difference with the `print-sigs` flag?

Reply:
It does not output the whole transaction, just the generated signature.

Reply:
Got the message thou, the flag is not very self-explanatory :)

I'm changing it into `--raw-signature`